Pop-Grupa-A-Frontend/src/services/nodeService.js

```diff
@@ -0,0 +1,36 @@
+import { NODE_URL } from '../commons/ApiLinks';
```

Comment:
na masterze całość operacji odbywa się w clusterService.js nie ma potrzeby robienie drugiego serwisu u nas już to wywalone jest.

Reply:
to jeszcze mój stary kod na potrzeby prezentacji tydzień temu więc można usunąć